unfoldingWord/translationCore
# Fix Dimmed Out FAB Buttons

#### This pull request addresses:
This PR fixes the import online and local import buttons being dimmed out and non-functional.


#### How to test this pull request:
Simply open the app and ensure the two icons next to the text for the import online and local are completely visible and functional.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3737)
